### ISSUE TITLE
Should we have more schema examples?

### ISSUE THREAD

**Issue by ePaul** [OAI/OpenAPI-Specification#1005]
I just noted that the 3.0-RC0 specification contains no examples where `oneOf` or `anyOf` is used (maybe also `not`, that is difficult to search).

Should we have some of them, as this is an new important feature?

**Comment by webron** [comment]
Not sure it would make sense. It would bloat the spec and it's not our job to teach people how to use JSON Schema. Maybe it makes more sense to add those in supporting documents.

**Comment by DavidBiesack** [comment]
I agree we don't want to turn the spec into a JSON Schema tutorial. the wiki would be a good place for such example.

**Comment by ePaul** [comment]
Okay. How can I contribute to the Wiki? I don't see any edit link (just a "clone locally" one).

**Comment by DavidBiesack** [comment]
@ePaul - perhaps this project is configured to not allow direct edits by rather requires [pull requests on the wiki repo](https://help.github.com/articles/adding-and-editing-wiki-pages-locally/). 

I suggest creating a **JSON Schema examples for OpenAPI 3.0** page or similarly named page.

**Comment by ePaul** [comment]
@DavidBiesack [There are no pull-requests for wiki repositories](http://stackoverflow.com/questions/10642928/how-to-pull-request-a-wiki-page-on-github) :-/

**Comment by webron** [comment]
As this is a documentation repo, I'd suggest staying away from the wiki and just work with md files in directories. It's a lot easier to manage and less error prone. The way the wiki works in github can be very destructive, and a maintenance hell.

**Comment by ePaul** [comment]
@webron should we incorporate the existing wiki pages there?

**Comment by wenjdu** [comment]
Giving you my 2 cents.
Having more examples would be really good !
I am really struggling trying to use oneOf, since the swagger-editor is showing errors.
